test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -135,6 +135,24 @@ class SkipTest(Exception):
 TIMEOUT_KEY = '__testcase_timeout__'
 
 
+def skip(reason=None):
+    """
+    Unconditionally skip a test.
+
+    :param reason: Reason for the skip
+    :type reason: str or None
+    """
+    skip_flag = True
+    if reason is None:
+        reason = "Skipped test execution"
```

Comment:
There is nothing wrong with this, but strings are immutable.  You can put it in the function header as a default argument.  You only have to worry about mutable types as default values.  Up to you.